src/features/game/events/landExpansion/fruitHarvested.ts

```diff
@@ -35,6 +35,26 @@ export function getFruitYield(name: FruitName, collectibles: Collectibles) {
   return 1;
 }
 
+function getHarvestedAt(
```

Comment:
In my opinion, we should reuse the getplantedAt function in fruitPlanted.ts file (Maybe change the name of the function to make it more generic). This will make sure that the growth time of the tree (initial growth + replenish time) always remains consistent between both actions. If in future we have more items which may give a boost to the growth time, it would reflect in both actions.